models/user.js

```diff
@@ -96,15 +95,14 @@ User.fromEvent = (event) => {
     }
   }
 
-  if (!isMobile) {
-    // Only set cookie/browser based fields when not a mobile event
-    // Android seemed to set these to random uuid's on each request.
-    forEach(browserFields, field => {
-      if (typeof event[field] !== 'undefined' && event[field]) {
-        user[field] = fieldValue(event[field], field)
-      }
-    })
-  }
+  // Skip network_userid field on mobile and server platforms.
+  // They seem to set these to random uuid's on each request.
+  const fields = includes(['mob', 'srv'], event.platform) ? without(browserFields, 'network_userid') : browserFields
```

Comment:
This looks like it will add `domain_userid` and `user_fingerprint` to mobile events now. Is that right?